apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/HelperFactoryImpl.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.shard.helper;
+
+import java.util.Optional;
+
+/**
+ * {@inheritDoc}
+ */
+public class HelperFactoryImpl implements HelperFactory<BatchedPaginationOperation> {
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Optional<BatchedPaginationOperation> createSelectInsertHelper(String helperTableName) {
```

Comment:
Maybe throw exception instead of using Optional?

Reply:
The throwing operation eats more CPU time, so optional is quite safe here.

Reply:
This factory is not called so frequent to affect CPU time when exception will be thrown. Exception will indicate, that `caller` code is broken will point us to it.

Reply:
added throw exception